geomesa-quickstart-kafka/src/main/java/org/geomesa/example/kafka/KafkaQuickStart.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0 which
+ * accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ */
+
+package org.geomesa.example.kafka;
+
+import com.vividsolutions.jts.geom.Envelope;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.geomesa.example.quickstart.GeoMesaQuickStart;
+import org.geomesa.example.quickstart.QuickStartData;
+import org.geomesa.example.quickstart.TDriveData;
+import org.geotools.data.DataStore;
+import org.geotools.data.DataStoreFinder;
+import org.geotools.data.DataUtilities;
+import org.geotools.data.Query;
+import org.geotools.data.collection.ListFeatureCollection;
+import org.geotools.data.simple.SimpleFeatureIterator;
+import org.geotools.data.simple.SimpleFeatureSource;
+import org.geotools.data.simple.SimpleFeatureStore;
+import org.geotools.feature.visitor.BoundsVisitor;
+import org.locationtech.geomesa.kafka.data.KafkaDataStoreFactory;
+import org.locationtech.geomesa.process.analytic.MinMaxProcess.MinMaxVisitor;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+import org.opengis.filter.Filter;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class KafkaQuickStart extends GeoMesaQuickStart {
+
+    // use t-dive streaming data
+    private QuickStartData data = new TDriveData();
+
+    private DataStore consumer = null;
+    private boolean wait = true;
+
+    public KafkaQuickStart(Map<String, String> params) {
+        super(params);
+    }
+
+    public KafkaQuickStart(String[] args) throws ParseException {
+        super(args, new KafkaDataStoreFactory().getParametersInfo());
+    }
+
+    @Override
+    public QuickStartData getData() {
+        return data;
+    }
+
+    @Override
+    @SuppressWarnings("AccessStaticViaInstance")
+    protected void addCustomOptions(Options options) {
+        options.addOption(Option.builder().longOpt("automated").build());
+    }
+
+    @Override
+    public DataStore createDataStore(Map<String, String> params) throws IOException {
+        System.out.println("Loading datastore");
+
+        // use geotools service loading to get a datastore instance
+        // we load two datastores - one is a producer, that writes features to kafka
+        // the second is a consumer, that reads them from kafka
+
+        params.put("kafka.consumer.count", "0");
+        DataStore producer = DataStoreFinder.getDataStore(params);
+        params.put("kafka.consumer.count", "1");
+        consumer = DataStoreFinder.getDataStore(params);
```

Comment:
`DataStore consumer = ...`  for completeness?

Reply:
it's a member variable already defined in the class - if I redefine it here it won't get set correctly